docs/InstrumentationVerification/InstrumentationVerification.md

```diff
@@ -30,7 +30,7 @@ To run a verification, simply:
 2. Run or restart the application we are instrumenting. Observe that as we instrument the app, more and more files will appear under [Datadog Logs Folder]/InstrumentationVerification/{ProcessName}_{ProcessID}_{ProcessCreationTime}
 3. Via command line, run the command: 
 
-    Datadog.InstrumentedAssemblyVerification.Standalone.exe [Full path of the folder generated in step 2] 
```

Comment:
These are two distinct options, right? So maybe break them apart?

Also, if I understand correctly, the option of using the full path to the folder that was generated in step (2), which you deleted, is the only way we support running the analysis on a process that already crashed, right? So perhaps we need to bring it back and explain that distinction?

Reply:
Fix in d6e34e581c70cc2271806c12db2c3d4f2caf78f1